pkg/cli/vault.go

```
@@ -30,8 +30,8 @@ const (
 
 	// VaultAWSKeychainNameDefault is the aws-vault default keychain name
 	VaultAWSKeychainNameDefault string = "login"
-	// VaultAWSProfileDefault is the aws-vault default profile name
-	VaultAWSProfileDefault string = "transcom-ppp"
+	// VaultAWSProfileTranscomPPP is the aws-vault profile name for transcom-ppp
+	VaultAWSProfileTranscomPPP string = "transcom-ppp"
```

Why `transcom-ppp` and not one of the new GovCloud profiles?

---

I'm not adding the GovCloud profiles because I'm not using the vault commands for these scripts anymore. I think in general we'll be pulling out vault commands from scripts for GovCloud because we can't use vault the same way, so I don't see the need to add those profiles here. Perhaps at some future point they'll be necessary.
This change is for the name of the ppp profile because I thought ProfileDefault was misleading at this point.

---

I can't imagine we would use `transcom-ppp` profile though, so it is possible to just get rid of it? `transcom-ppp` is no more.

---

I don't know what pulling that out would impact for other scripts. I would like to leave that to later in the commercial tear down process as it seems auxiliary to this work.